Projects/Foundations/Schemas/Docs-Nodes/C/Crypto/Crypto-Exchange/crypto-exchange.json

```diff
@@ -168,34 +168,35 @@
         },
         {
             "style": "List",
-            "text": "maxRate sets the maximum number of OHCLV that is pulled before the data is saved. This is only to be used when the exchange is kicking out the data-mine randomly and alows the user to save the data more often allowing for the data mining to move forward.",
+            "text": "maxRate sets the maximum number of OHCLV that is pulled before the data is saved. This is only to be used when the exchange is kicking out the data-mine randomly and to allow to the user to save the data more often allowing for the data mining to move forward.",
             "translations": [
                 {
                     "language": "RU",
                     "text": "maxRate устанавливает максимальное количество OHCLV, которое извлекается перед сохранением данных. Это должно использоваться только тогда, когда биржа случайным образом запускает извлчение данных и позволяет пользователю сохранять данные чаще, что позволяет продвигать интеллектуальный анализ данных.",
                     "updated": 1638648664286
                 }
-            ]
+            ],
+            "updated": 1650806325108
```

Comment:
No, I did nothing manually. All done in the SA inteface

Reply:
Check the screenshot. I think that comma is there because of the new line after that. 
![image](https://user-images.githubusercontent.com/38046064/165911628-dad8c270-3201-4033-a6c4-79f1ef04d719.png)